ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb

```diff
@@ -8,18 +8,9 @@
     "\n",
```

Comment:
<p>Notes:</p><ol><li>Tell customers to, if using a private workforce, replace <code>&lt;&lt;ADD WORK TEAM ARN HERE&gt;&gt;</code>  with the work team ARN in the cell below so they don't miss that step. </li><li>Add period after "Create and Manage Workforces"</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>